content: update theme (#24)
jsheroes/website: src/content/theme.md

## 10 years of pushing the boundaries

> This year we want to shift our attention to the decade of progress in the world of JavaScript. Back in 2013, we went through a period of rethinking practices as frontend development took off and became one of the most common disciplines in tech. Framework ecosystems like React, Angular or Vue now enter into their second decade of existence, while web capabilities and standards are increasingly more potent and versatile.

Reviewer: ```suggestion
This year we want to shift our attention to the decade of progress in the world of web development. Back in 2013, we went through a period of rethinking practices as frontend development took off and became one of the most crucial disciplines in tech. Framework like React, Angular or Vue and their ecosystems now enter into their second decade of existence, while web capabilities and standards are increasingly more potent and versatile.
```

Author: Love the `crucial` vs `common`, great idea!

So in 2023, let's celebrate the past 10 years, reflecting on the evolution of our tools and patterns, with the good and the bad that comes with this rapid evolution.

> At JSHeroes, we always ventured outside the pure language ecosystem, so expect to see a variety of talks that cover the entire landscape of Web Development as well as JavaScript as a universal programming language. But of course, we will not neglect the human side of things, in our effort to paint a full picture for the development community in 2023.

Reviewer: ```suggestion
At JSHeroes, we always ventured outside the pure language ecosystem, so expect to see a variety of talks covering the entire landscape of web development and JavaScript as a universal programming language.

But of course, we will not neglect the human side of things, in our effort to paint a full picture for the development community in 2023.
```
